meta/lib/oeqa/selftest/cases/devtool.py

```diff
@@ -2328,12 +2328,11 @@ def test_devtool_ide_recipe_cmake(self):
 
             # Verify re-building and testing works again
             result = runCmd('%s --build --preset %s --target clean' % (cmake_exe, preset_name), cwd=tempdir)
-            self.assertIn("Cleaning all built files...", result.output)
```

Comment:
Not sure what's the issue. I just rebased my patches to the latest master and the tests pass without this patch here.

Reply:
On my end I can confirm that the cmake commands use a different output on my seutp.I'm getting:
```
2023-10-09 10:16:57,130 - oe-selftest - INFO - Traceback (most recent call last):
  File "/home/deribaucourt/Workspace/yocto-vscode/yocto/yocto-build/sources/poky/meta/lib/oeqa/selftest/cases/devtool.py", line 2337, in test_devtool_ide_recipe_cmake
    self.assertIn("Cleaning all built files...", result.output)
AssertionError: 'Cleaning all built files...' not found in "Change Dir: '/home/deribaucourt/Workspace/yocto-vscode/yocto/yocto-build/build-st/tmp/work/core2-64-poky-linux/cmake-example/1.0/cmake-example-1.0'\n\nRun Build Command(s): ninja -v clean\n[1/1] ninja  -t clean \nCleaning... 8 files."
```
My workspace has plain poky nanbield, with
 - cmake: 3.27.5
 - ninja: 1.11.1

Reply:
The tests are changed more or less as you suggested. The patches are rebased to todays master and they pass.